tutor/src/components/my-courses/listings.jsx

```diff
@@ -46,7 +46,17 @@ function MyCoursesNone() {
   );
 }
 
-const MyCoursesCreate = () => wrapCourseItem(CreateACourse);
+//TODO: Adding a default 'intro_sociology' for now to skip selecting an offering when creating a new course.
+const MyCoursesCreate = () => {
+  return (
+    <Col lg={3} md={4} sm={6} xs={12}>
+      <CreateACourse
+        appearanceCode="intro_sociology"
```

Comment:
Just double checking that this should be here?

Reply:
it shouldn't, I just had it as a placeholder.